bin/faucet/src/main.rs

```diff
@@ -130,17 +131,21 @@ pub enum Command {
         #[clap(flatten)]
         config: ClientConfig,
 
-        /// URL to bind the API server.
-        #[arg(long = "api-bind-url", value_name = "URL", env = ENV_API_BIND_URL)]
-        api_bind_url: Url,
+        /// Port to bind the API server.
+        #[arg(long = "api-bind-port", value_name = "PORT", env = ENV_API_BIND_PORT, default_value = "8000")]
+        api_bind_port: u16,
```

Comment:
nit: should we add to the comment that the server will be started on `0.0.0.0`? Something like:

```
/// Port to bind the API server. The server will be started on `0.0.0.0:api-bind-port`.
```